src/org/labkey/test/components/react/BaseReactSelect.java

```diff
@@ -541,7 +541,9 @@ public BaseReactSelectFinder<Select> withContainerClass(String containerClass)
 
         public BaseReactSelectFinder<Select> withId(String id)
         {
-            _locator = Locators.containerWithDescendant(Locator.tagWithId("div", id));
+            Locator.XPathLocator idElementLoc = Locator.XPathLocator.union(
+                    Locator.tagWithId("div", id), Locator.tagWithId("input", id));
+            _locator = Locators.containerWithDescendant(idElementLoc);
```

Comment:
Not sure this needs to care about the tag at all
```suggestion
            _locator = Locators.containerWithDescendant(Locator.id(id));
```